src/components/Location/index.js

```diff
@@ -22,48 +14,35 @@ const IFrame = styled.iframe`
   height: 300px;
 `
 
-const LocationBlock = ({ index, name, location, googleMapUrl, imageList }) => (
-  <Accordion elevation={0} defaultExpanded={index === 0}>
-    <AccordionSummary
-      expandIcon={<ExpandMoreIcon />}
-      aria-controls={`panel-${index}-content`}
-      id={`panel-${index}-header`}
-    >
-      <Typography variant="h5" gutterBottom>
+const Title = styled(Typography)`
+  color: ${props => props.theme.color.green};
+`
+
+const LocationBlock = ({ name, location, googleMapUrl, imageList }) => (
+  <Grid container spacing={5}>
+    <Grid item xs={12} sm={5}>
+      <Title variant="h5" gutterBottom>
         {name}
+      </Title>
+      <Typography variant="body2" gutterBottom>
+        {location}
       </Typography>
-    </AccordionSummary>
-    <AccordionDetails>
-      <Grid container spacing={5}>
-        <Grid item xs={12} sm={5}>
-          <Typography variant="body2" gutterBottom>
-            {location}
-          </Typography>
-          <IFrame src={googleMapUrl} title={name} />
-        </Grid>
-        <Grid item xs={12} sm={7}>
-          <Carousel
-            showStatus={false}
-            dynamicHeight={true}
-            autoPlay={true}
-            infiniteLoop={true}
-          >
-            {imageList.map((img, index) => (
-              <StdImage key={index} imageObj={img} />
-            ))}
-          </Carousel>
-        </Grid>
-      </Grid>
-    </AccordionDetails>
-  </Accordion>
+      <br />
+      <IFrame src={googleMapUrl} title={name} />
+    </Grid>
+    <Grid item xs={12} sm={7}>
+      <Carousel
+        showStatus={false}
+        dynamicHeight={true}
+        autoPlay={true}
+        infiniteLoop={true}
```

Comment:
I think you don't need `{true}` in jsx
```suggestion
        dynamicHeight
        autoPlay
        infiniteLoop
```
should be good enough.

Reply:
true.

Reply:
You know you could commit my suggestions together directly in github😉

Reply:
WTF how